libs/widget-lib/src/types.ts

```diff
@@ -101,7 +100,12 @@ interface CowSwapWidgetConfig {
    */
   chainId: SupportedChainId
   /**
-   * The token lists to use in the widget. Default: CoW Protocol token list.
+   * When enabled, the widget will only use the token lists provided in the `tokenLists` property.
+   * Otherwise, it will append the provided token lists to default ones.
+   */
+  useOnlyProvidedTokenLists: boolean
```

Comment:
why not making the interface simpler? `tokenLists`, if not provided or 0 items, then we use the default, otherwise we use the provided list.

Isn't this enough and simpler?

Reply:
I think there might be a case, when a consumer would want to append token lists instead of replacing them.
But anyway, this is only a hypothesis, so I will remove it.